Auto fill exceptions from alert data (#3544)
elastic/security-docs: docs/detections/add-exceptions.asciidoc

.. Click *Create shared exception list* -> *Create exception item*. 
--

. In the *Add rule exception* flyout, do the following: 

.. Name the exception. 

> .. Add conditions that define the exception. When the exception's query conditions are met (the query evaluates to `true`), rules do not generate alerts even when other rule criteria are met.

Reviewer: I had a little difficulty following the second sentence -- there's a lot of passive voice, maybe could be streamlined. I also thought "other rule criteria" sounded a little vague, like maybe some rule criteria are involved while other criteria might not be, but I think it's just the rule itself.
```suggestion
.. Add conditions that define the exception. When the exception's query evaluates to `true`, rules don't generate alerts even when their criteria are met.
```

+

> NOTE: Exception conditions are auto-populated with relevant alert data when you create a new exception from the alert's details flyout or from the Alert table.

Reviewer 2: Great description Nastasha!.
It is indeed accurate that there is no need to mention the "Highlighted fields" since we only utilize the relevant data from them only.

Author: Note to self that I might need to expand this note to include the comment that's also auto-generated when users create a new exception from the Alerts table or flyout.

Author: ```suggestion
NOTE: When you create a new exception from the alert's details flyout or from the Alert table, exception conditions are auto-populated with relevant alert data. A comment describing this action is also automatically added to the **Add comments** section.
```

> NOTE: Exception conditions are auto-populated with relevant alert data when you create a new exception from the alert's details flyout or from the Alert table.

Author: Thank you! I added your feedback in [059a805](https://github.com/elastic/security-docs/pull/3544/commits/059a805dedf498622f1f7bb9fdb15b43a51c765c).

> NOTE: When you create a new exception from the alert's details flyout or from the Alert table, exception conditions are auto-populated with relevant alert data. A comment describing this action is also automatically added to the **Add comments** section.

Reviewer: We could probably simplify this -- I get that naming each particular context adds specificity, but it gets the sentence off to a long, slow start, and users might not need to know every context (X and Y parts of the UI), just the general context (alerts).
```suggestion
NOTE: When you create a new exception from an alert, exception conditions are auto-populated with relevant alert data. A comment describing this action is also automatically added to the **Add comments** section.
```

Author: Yeah good point, especially when the necessary context is provided in the previous step. I'll commit this edit - thanks!
